applications/bountyblok.md

```diff
@@ -0,0 +1,184 @@
+
+- What need(s) does your project meet?
+  It is a free tool which allows projects on EOS to help them grow their community and user-base 
+
+- Are there any other projects similar to yours in the EOSIO ecosystem?
+  Not that I am aware
+  
+   
+## Team
+
+We are mostly a tech-strong group of guys based in Montreal, Canada
+You can view our team on slide #3 https://bountyblok.io/deck.pdf
+
+### Team members
+
+* Dimitri Nikolaros: Software Developer/Team Lead, holds B.Eng from Concordia University in Computer Engineering.
+* Jan Bui: Software Architect, holds a B.Comm from McGill University in Management Information System and Accountancy.
+* Saddam Tahir: Software Developer, holds a Masters from Concordia University in Computer Science.
+* Noman Mirza: Software Developer, holds a Masters from Concordia University in Computer Science .
+* Jackie Chao: Software Developer, holds B.Eng from Concordia University in Software Engineering.
+* Mike Barwick: UX/UI Designer
+
+### Contact
+
+* **Contact Name:** Dimitri Nikolaros
+* **Contact Email:** dimitri.nikolaros@inlinefx.com
+* **Website:** https://eos.drop.bountyblok.io
+
+### Legal Structure
+
+* **Registered Address:** 3770 rue Jean-Paul-Sartre Laval (Québec) H7P0E4 Canada
+* **Registered Legal Entity:** inlineFX Software (9278-3521 Québec inc.)
+
+### Team Experience
+
+The bountyblok team carries a strong technical background in various areas of engineering from front end to backend development, EOSIO and various blockchain technologies. 
+
+* Jan Bui:
+
+  With over 2 decades' experience in technical leadership and a sound vision of the delicate balance between technology and business, Jan has positioned inlineFX at the forefront of the technology landscape. He has worked previously in Montreal, Zurich and London, working and managing high-level teams of front-office software developers at various banks and brokerages such as Credit Suisse, Morgan Stanley and BGC Partners.
+
+* Dimitri Nikolaros:
+
+  As Co-Founder and lead programmer, Dimitri focuses on delivering quality and functional software. He has developed websites that have processed millions of transactions and currently used by thousands of avid users. In addition, Dimitri focuses on client-relationship management and other strategic initiatives.
+
+### Team Member Repos
+
+* https://github.com/bountyblok 
+* https://github.com/diminiko
+
+### Team LinkedIn Profiles (if available)
+
+* https://www.linkedin.com/in/dnikolaros/ Dimitri Nikolaros
+* https://www.linkedin.com/in/jan-bui-4b62383/ Jan Bui
+
+## Development Status
+
+The distribution tool went live on EOS in Q4 2021: https://eos.drop.bountyblok.io. However, this version went live with a limited set of features as we required additional infrastructure at the time.
+
+## Development Roadmap
+
+### Overview
+
+We are looking to expand our features and offer what we have right now on the WAX version and some of these features include:
```

Comment:
This proposal is worded ambiguously regarding whether all of these listed features are available on the version available on WAX and simply need to be ported over to be available on EOS, or if some of these are new features that need to be developed.

It would be helpful to understand which are specifically the new features to be developed, what type of work is needed to implement that feature (frontend, backend, smart contract, etc.), and how that fits into the milestones below to get a better understanding of how the estimated time and costs reflect the work to be done.

And to better understand the work required for porting features already available on WAX, it will be helpful to understand what are the significant differences between the two platforms as it relates to this tool which necessitate considerable additional work. I assume a lot of that is captured with infrastructural deliverables captured in milestone 1, but I would prefer to confirm.

Reply:
Hi Areg,

It is essentially a port over to to EOS

- new frontend
- new backend
- SHIP node for EOS specific 
- atomicassets DB for EOS NFTs
- wallet integration (Anchor)

Things that are new:
- wallet integration with Wombat Wallet
- implementation to handle EOS endpoints, EOS validation (for example our parser for WAX verifies .wam wallets which is not the case for EOS), we anticipate from experience several changes like these that differ from WAX

It is essentially porting over the product to EOS and keeping the UX very similar because projects are used to doing things in a certain way on WAX and we would like to maintain that UX.

Thanks